one_collect/src/helpers/exporting/scripting.rs

```diff
@@ -1000,9 +1000,14 @@ impl ScriptedUniversalExporter {
     pub fn from_script(
         self,
         script: &str) -> anyhow::Result<UniversalExporter> {
+        info!("Starting script execution for export configuration");
```

Comment:
These are good, but I would like much more detail during script parsing and construction.